htdocs/web_portal/index.php

```diff
@@ -84,6 +86,18 @@ function rejectIfNotAuthenticated($message = null){
 try {
     Draw_Page($Page_Type);
 
+} catch (BadCredentialsException $error) {
+    /**
+     * `show_view('error.php', ...` is not suitable here.
+     * - setting raw to FALSE triggers another exception because it tries
```

Comment:
What does `raw` refer to?

Reply:
It's one of the arguments of [show_view](https://github.com/gregcorbett/gocdb/blob/e1c9d7751cbf26d8a4528291f39035f222d5d28c/htdocs/web_portal/index.php#L731)

Reply:
Isn't that `rawOutput`?

Reply:
ah yes, d'oh 😝